src/app/modules/ix-table/components/ix-table-columns-selector/ix-table-columns-selector.component.ts

```diff
@@ -80,6 +80,8 @@ export class IxTableColumnsSelectorComponent<T = unknown> implements OnChanges,
       untilDestroyed(this),
     ).subscribe((displayedColumns) => {
       this.columns().forEach((column) => {
+        if ('actions' in column) return;
```

Comment:
Maybe extract `actions` in a constant to use here and in other places.

Reply:
> Maybe extract `actions` in a constant to use here and in other places.

@undsoft I don't see other places where it's used

Reply:
Well, there is some connection to the `IxCellActionsComponent`. It would be good to make sure it's not fragile. For example if you can compare type to `IxCellActionsComponent` that may be better than checking for a key.

Reply:
@undsoft, yeah - sounds good.

Gonna use `if (column instanceof IxCellActionsComponent) return;`